source/_data/plugins.yml

```diff
@@ -1557,3 +1557,15 @@
     - excerpt
     - abstract
     - description
+- name: hexo-codepen
+  description: A better version of hexo-tag-codepen with a valid HTML.
+  link: https://github.com/maliMirkec/hexo-tag-codepen
+  tags:
+    - tag
```

Comment:
It is new [Hexo tag](https://hexo.io/docs/tag-plugins.html) for codepen. You may change this if unnecessary.